tests/timecode.test/Timecode_Test.cs

```diff
@@ -377,6 +377,94 @@ public void MoreThan_Timecodes_With_Different_Framerates()
 
       // Assert
       comparison.Should().Throw<InvalidOperationException>();
+    }
+
+    #region staticmethodtests
+

```

Comment:
As we will implement a new static method we should add tests for it as well. I will suggest tests below:

```suggestion
[Fact]
    public void Convert_FrameRate_Using_Static_Method()
    {
      var sut = Timecode.ConvertFramerate("10:00:00:00", Enums.Framerate.fps23_976, Enums.Framerate.fps59_94_NDF);
      sut.Should().Be("04:00:00:00");
    }

```

Reply:
@RasmusBroborg Above review comments applied and committed the changes